extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -63,7 +64,8 @@ require([
 			willAutoplay = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries,
 			autoplayOnLoad = willAutoplay && !document.hidden,
 			initialPlayTriggered = false,
-			recommendedVideoDepth = 0;
+			recommendedVideoDepth = 0,
+			logGroup = 'article-featured-video';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'logGroup' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "logGroup" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)